manager/src/main/resources/db/migration/0.1/V0.1.23__add_bridge_error_info.sql

```diff
@@ -0,0 +1,8 @@
+-- add new "error_code" column
+alter table BRIDGE
+    add column "bridge_error_id" integer,
```

Comment:
I think the comment in the `.sql` is a bit misleading. I also personally think `error_id` and `error_uuid` would be OK as column names.

Reply:
Fixed.